athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/data/SupportedTypes.java

```diff
@@ -36,6 +36,7 @@
     BIT(Types.MinorType.BIT),
     DATEMILLI(Types.MinorType.DATEMILLI),
     DATEDAY(Types.MinorType.DATEDAY),
+    TIMESTAMPMILLI(Types.MinorType.TIMESTAMPMILLI),
```

Comment:
i thought the task was to add support for TIMESTAMPMILLITZ, wont it be weird if we add TIMESTAMPMILLI (no TZ) and tell people it has TZ awareness? What am i missing